nautobot/extras/utils.py

```diff
@@ -219,3 +219,42 @@ def refresh_job_model_from_job_class(job_model_class, job_source, job_class, *,
     )
 
     return (job_model, created)
+
+
+def validate_webhooks(instance, content_types, payload_url, type_create, type_update, type_delete):
```

Comment:
Any thoughts on making this a method on the `Webhook` class itself versus a standalone utility function? Maybe something like `Webhook.check_for_conflicts(self, ...)`?